velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -37,6 +37,22 @@ bool isParquetReservedKeyword(
       ? true
       : false;
 }
+
+// An unannotated array in Parquet is a repeated field that is not explicitly
+// marked as a LIST logical type, and it depends on the reader to decide
+// whether the data should be interpreted as scalar values or array. If current
```

Comment:
@rui-mo I don't think an unannotated array in Parquet with a `repeated` field can be read as scalar values.
Can you clarify this comment?